src/bun.js/node/node_util_binding.zig

```diff
@@ -194,3 +189,13 @@ pub fn SplitNewlineIterator(comptime T: type) type {
         }
     };
 }
+
+pub fn normalizeEncoding(globalThis: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) bun.JSError!JSC.JSValue {
```

Comment:
this should probably be done in C++ so we can reuse encoding strings instead of creating a new encoding string every time